jdaviz/configs/default/plugins/viewers.py

```diff
@@ -198,7 +198,11 @@ def _expected_subset_layer_default(self, layer_state):
             return
 
         # default visibility based on the visibility of the "parent" data layer
-        layer_state.visible = self._get_layer(layer_state.layer.data.label).visible
+        if self.__class__.__name__ != 'RampvizProfileView':
+            layer_state.visible = self._get_layer(layer_state.layer.data.label).visible
+        else:
+            # Rampviz doesn't show subset profiles by default:
+            layer_state.visible = False
```

Comment:
this else applies to more than just rampviz though, right?

Reply:
I'm not sure what you mean - the logic introduced here only affects Rampviz.

Reply:
yes, but this is in the general `JdavizViewerMixin` used by viewers across all configs, so the `else` will be entered by all other cases than `RampvizProfileView`.  For non-rampviz cases, we need the previous behavior of adopting visibility from the corresponding parent data layer.

Reply:
The above `if` is:
```python
        if self.__class__.__name__ != 'RampvizProfileView':
```

Reply:
ah, I misread that as `==` 🤦  and further confused because the comment in the else mentions rampviz.  Similar confusion might be avoided by switching the if and else so the else is the default case (or just paying more attention on my part).

Reply:
both are true. I'll switch :D